ovos_PHAL_plugin_commonIOT/device_manager.py

```diff
@@ -73,6 +171,8 @@ def load_scanners(self):
             scanner.start()
             self.scanners[plugin] = scanner
 
+    def get_device(self, device):
+        return self.devices.get(device.device_id, None)
```

Comment:
this will need more advanced matching later, we dont get a device_id, we get a human query that needs to be matched against all registered devices